lib/Targeting/targeting_manager.dart

```diff
@@ -0,0 +1,206 @@
+      case FSOperator.LOWER_THAN:
+        return isCurrentValueIsLowerThanAudience(cuurentValue, audienceValue);
+      case FSOperator.LOWER_THAN_OR_EQUALS:
+        return isCurrentValueIsLowerThanOrEqualAudience(cuurentValue, audienceValue);
+      case FSOperator.CONTAINS:
+        return isCurrentValueContainAudience(cuurentValue, audienceValue);
+      case FSOperator.NOT_CONTAINS:
+        return !isCurrentValueContainAudience(cuurentValue, audienceValue);
+      default:
+        return false;
+    }
+  }
+
+  /// Compare EQUALS
+  bool isCurrentValueEqualToAudienceValue(dynamic currentValue, dynamic audienceValue) {
+    return (currentValue == audienceValue);
+  }
+
+  /// Compare greater than
+  bool isCurrentValueIsGreaterThanAudience(dynamic currentValue, dynamic audienceValue) {
+    if (currentValue is num && audienceValue is num) {
+      return (currentValue > audienceValue);
+    } else if (currentValue is String && audienceValue is String) {
+      return (currentValue.compareTo(audienceValue) == 1);
+    } else {
+      return false;
+    }
+  }
+
+  /// Compare greater than or equal
+  bool isCurrentValueIsGreaterThanOrEqualAudience(dynamic currentValue, dynamic audienceValue) {
+    if (currentValue is num && audienceValue is num) {
+      return (currentValue >= audienceValue);
+    } else if (currentValue is String && audienceValue is String) {
+      return (currentValue.compareTo(audienceValue) == 1 || currentValue.compareTo(audienceValue) == 0);
+    } else {
+      return false;
+    }
+  }
+
+  /// Compare lower than
+  bool isCurrentValueIsLowerThanAudience(dynamic currentValue, dynamic audienceValue) {
+    if (currentValue is num && audienceValue is num) {
+      return (currentValue < audienceValue);
+    } else if (currentValue is String && audienceValue is String) {
+      return (currentValue.compareTo(audienceValue) == -1);
+    } else {
+      return false;
+    }
+  }
+
+  /// Compare lower than or equal
+  bool isCurrentValueIsLowerThanOrEqualAudience(dynamic currentValue, dynamic audienceValue) {
+    if (currentValue is num && audienceValue is num) {
+      return (currentValue <= audienceValue);
+    } else if (currentValue is String && audienceValue is String) {
+      return (currentValue.compareTo(audienceValue) == -1 || currentValue.compareTo(audienceValue) == 0);
+    } else {
+      return false;
+    }
+  }
+
+  /// Compare contain
+  bool isCurrentValueContainAudience(dynamic currentValue, dynamic audienceValue) {
+    if (currentValue is String && audienceValue is String) {
+      return (currentValue).contains(audienceValue);
+    }
+    return false;
+  }
+
+  dynamic getCurrentValueFromCtx(String targetKey) {
+    if (targetKey == FS_USERS) {
+      return userId;
+    } else {
+      return currentContext[targetKey];
+    }
+  }
+
+  static FSOperator createOperator(String raw) {
+    if (raw == 'EQUALS') {
```

Comment:
I think something like that is possible, no:
```
for (var value in FSOperator.values){
  if (value.toString() === raw){
    return value
  }
}
return FSOperator.Unknown
```

Reply:
I which is possible , because .toString give us : "FSOperator.EQUALS"

Reply:
You can get the last part like this: `value.toString().toString().split('.').last`
But if you prefer the if I understand ;)